Makefile

```diff
@@ -202,6 +202,11 @@ format:
 	@./tools/format_all.sh
 	@make --no-print-directory -C test format
 
+.PHONY: typofix
+typofix:
```

Comment:
Don't need a Makefile target for typo fixing. Just like we don't have a magic button or Makefile target that fixes compiler, lint, or style errors, either.